suite/src/__tests__/fast/composedb-queries.test.ts

```diff
@@ -0,0 +1,129 @@
+import { beforeAll, describe, expect, test} from '@jest/globals'
+import * as testHelpers from '../../utils/composeDBHelpers.js'
+import { ComposeClient } from '@composedb/client'
+
+//const ComposeDbUrls = String(process.env.COMPOSEDB_URLS).split(',')
+const ComposeDbUrls = ['http://localhost:7007']
+///////////////////////////////////////////////////////////////////////////////
+/// Create/Update/Queries ComposeDB Models Tests
+///////////////////////////////////////////////////////////////////////////////
+
+describe('create/update/queries on ComposeDB Models', () => {
+    let id: string
+    let compose: ComposeClient;
+    const originalText = `Sample Text was created at ${Date.now().toString()}`
+    const updatedText = originalText + " and updated"
+    const numValue = Math.floor((Math.random() * 100) + 1) // Random number between 1 and 100
+    const updatedNumValue = numValue + 1 
+    const boolValue = true;
+
+    beforeAll(async () => {
+        compose = await testHelpers.setUpEnvironment(ComposeDbUrls[0])
+    })
+
+    test('test create', async () => {
+      try {
+        const response = await compose.executeQuery(`mutation {
+          createGenericModel(input: {
+                  content: {
+                      numericalField: ${numValue},
+                      textField: "${originalText}",
+                      booleanField: ${boolValue}
+                  }
+              }) 
+              {
+                  document {
+                      id
+                      numericalField
+                      textField
+                      booleanField
+                    }
+              }
+        }`)
+        console.log(`Executing a create mutation for the GenericModel with textField: "${originalText}"`)
+        expect(response.errors).toBeUndefined()
+        expect(response.data).not.toBeUndefined()
+        expect(response.data?.createGenericModel).not.toBeUndefined()
+        const record = (response.data?.createGenericModel as any).document
+        expect(record).not.toBeUndefined()
+        // save the recently created record id
+        id = record.id
+        expect(id).not.toBeNull();
+        expect(id).not.toBeUndefined();
+        expect(id.length).toBe(63)
+        expect(record.numericalField).toBe(numValue)
+        expect(record.textField).toBe(originalText)
+        expect(record.booleanField).toBe(boolValue)
+       } catch (err) {
```

Comment:
Oh, I see. I was under the impression that because other test files used them they were mandatory.

Reply:
I think we've added try/catch to a few specific tests that were having failures in CI and we wanted to log additional information on failure about the state of the system to help us debug what was going wrong.  Since these log messages don't really add any new information, I think we can omit them for now.

Reply:
This is ready!